### PR TITLE
Sds/Services, Products, Open Source Page

### PR DESCRIPTION
**Pull request by Arevyn24**
### Add new page with three new sections

To Do:
- [x] Pick new icons for all the things
- [x] Add parallax images
- [x] Add descriptions to services and anything else with missing text
- [x] Remove "Team Growth" and "Coming soon" from current products section
- [x] Create a second products section with "Team Growth" and "Coming Soon" listed

### CODE REVIEW

**Comment by Arevyn24** [comment]
Copy-pasta'd services section (still needs a lot of work)
image

**Comment by Arevyn24** [comment]
### New Products Section (on side page)
image

### Open Source Section
image

File: source/_open_source.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<div class="open_source container main-colors" id="open_source">
+<h4 class="section-title">Open Source Software</h4>
+  <div class="row">
+  	<div class="col s12 m6">
+      <div class="icon-block">
+        <h2 class="center"><i class="material-icons md-48"><img src="images/chair-school.png"></i></h2>
+        <h4 class="center">
+          <a href="https://github.com/SavvySoftWorksLLC/body-snatchers">Brown Noser</a>
```
Comment (garyanewsome) [inline source/_open_source.html.erb]: We will need to link to the `Brown Noser` repo, I thought we had one in SSW, we will have to ask Paul

Comment (ninjapanzer) [inline source/_open_source.html.erb]: Its in https://github.com/WCCCEDU/brown_noser_rb

**Comment by Arevyn24** [comment]
#### Services Section
 image
image

#### Products Section
image

#### Open Source Section
image

**Pull request merged**